symbiote/silverstripe-multivaluefield/SS-2018-017-1.yaml

```
@@ -0,0 +1,8 @@
+title:     'SS-2018-017: Possible PHP Object Injection via Multi-Value Field Extension'
+link:      https://www.silverstripe.org/download/security-releases/ss-2018-017/
+cve:       ~
+branches:
+    3.0.x:
+        time:     2018-07-16 17:29:04
+        versions: ['>=3.0.0', '<3.0.99']
```

```suggestion
        versions: ['>=3.0.0', '<3.1.0']
```

Gain, would this be more accurate?

---

might just leave this one, most recent one is 3.0.5, remote chance it'll make its way up to 3.0.9.  I don't want to use <3.1.0 style because then it won't match how we've done it for all the other minors